Simple Inventory Management System/Inventory.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Simple_Inventory_Management_System
+{
+    class Inventory
+    {
+        private List<Product> products = new List<Product>();
+
+        public void AddProduct(string name, decimal price, int quantity)
+        {
+            Product product = new Product { Name = name, Price = price, Quantity = quantity };
+            products.Add(product);
+            Console.ForegroundColor = ConsoleColor.Green;
+            Console.WriteLine("Product added successfully.");
+            Console.ForegroundColor = ConsoleColor.White;
```

Comment:
This method do two things, adding a product and display a success message. It is good to separate the logic, either you move it to the main class or another appropriate class.